site/src/components/keyboard-controls/KeyboardControl.tsx

```diff
@@ -34,7 +36,11 @@ const Keys: FC<{ keyOrCombos: string }> = ({ keyOrCombos }) => {
         if (index % 2 === 1) {
           return <Fragment key={index}> {keyOrSeparator} </Fragment>;
         }
-        return <kbd key={index}>{keyOrSeparator}</kbd>;
+        return (
+          <kbd key={index} className={styles.key}>
```

Comment:
Nit: Is it worth adding a classname here? I doubt we would want the `<kbd />` element to be styled differently elsewhere so we probably can just use that as a selector

Reply:
using the selector now